data-access/basic_data_access.ipynb

```diff
@@ -16,21 +39,20 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import climakitae as ck \n",
-    "from climakitae.core.data_interface import (\n",
-    "    get_data_options, \n",
-    "    get_subsetting_options, \n",
-    "    get_data\n",
-    ")"
+    "import pandas as pd\n",
+    "import xarray as xr\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import climakitae as ck "
    ]
   },
   {
    "cell_type": "markdown",
    "id": "588887a8",
    "metadata": {},
    "source": [
-    "## High-level details \n",
-    "The AE data catalog has many different types of data. Our helper library `climakitae` attempts to make accessing and retrieveing this data intuitive, as well as simplify climate and statistical analysis with the data down the line, by performing some data transformations as the data is retrieved.<br><br> To retrieve the data, you'll need to make some selections as to your climate variable, data resolution, location settings, and many other options. There are also several high-level options you'll need to set when selecting your data, detailed below: \n",
+    "# High-level details \n",
```

Comment:
Introduce `ClimateData` in this section as the user interface that consolidates ALL data access / processing methods from `climakitae`